vars/pragmasToEnv.groovy

```diff
@@ -25,6 +25,9 @@ String call(String commit_message) {
         }
     }
 
+    // put the pragms into the environment
+    env.pragmas = pragmas
```

Comment:
@JohnMalmberg Does it surprise you that this works?  It should, because the best I can determine, it doesn't, unfortunately.  My testing of this was flawed.  :frowning:  It doesn't seem that a shared-lib function can modify the environment like this.  Unless you know otherwise.

Are you aware of any way to achieve what this is trying to achieve?

Reply:
I take that back.  It DOES work!

My original test was successful.  It was a subsequent test that was failing and that's probably due to a bug.  So it was useful testing in the end.